community/shellinabox/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=shellinabox
+pkgver=3.9.6
+pkgrel=0
+pkgdesc="publish command line shell through AJAX interface"
+url="https://github.com/shellinabox/shellinabox"
+arch="x86_64"
+license="GPL-2.0"
+depends="zlib musl"
+makedepends="libressl-dev linux-pam-dev zlib-dev autoconf automake libtool g++"
+install="$pkgname.post-install"
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/shellinabox/shellinabox/archive/master.tar.gz launcher.patch"
+builddir="$srcdir"/shellinabox-master
+
+
+prepare() {
+	patch -p0 -i "$srcdir"/launcher.patch
+}
+
+build() {
+	cd "$builddir"
+	autoreconf -i
+	./configure
+	make
+}
+
+check() {
+	cd "$builddir"
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+
+	# Avoid writing on /usr/local so we fix it here
+	mv "$pkgdir"/usr/local/bin "$pkgdir"/usr/bin
+	mv "$pkgdir"/usr/local/share "$pkgdir"/usr/share
+	rmdir "$pkgdir"/usr/local 
+
+	# Compress man file
+	gzip -c9 "$pkgdir"/usr/share/man/man1/shellinaboxd.1 > "$pkgdir"/usr/share/man/man1/shellinaboxd.1.gz 
+	rm "$pkgdir"/usr/share/man/man1/shellinaboxd.1
+
```

Comment:
I am not sure if this is necessary. `subpackages="$pkgname-doc"` is automatically handling man pages.

Reply:
I got an error while building saying that man file is not compressed.

Reply:
You could also patch the function that generated the man page https://github.com/shellinabox/shellinabox/blob/98e6eebc6c2026fb126a458c6cb5a2541447258e/Makefile.am#L369